ga4gh/frontend.py

```diff
@@ -93,6 +93,10 @@ def handleHttpOptions():
 
 @app.errorhandler(Exception)
 def handleException(exception):
```

Comment:
Yeah, the trickiness may be in part because of the complexity of the model we have.  I'll add some comments.